test/test-execute/exec-standardoutput-file.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Test for StandardOutput=file:
```

Comment:
Because that’s the syntax to direct stdout to a file? I was generally trying to follow the existing `exec-standardinput-file.service` test here (that’s also why I had the absolute paths below).